src/openapi.yaml

```diff
@@ -3763,6 +3755,162 @@ paths:
         - zds.scopes.zaken.bijwerken
       requestBody:
         $ref: '#/components/requestBodies/Zaak'
+    delete:
+      operationId: zaak_delete
+      description: Verwijdert een zaak, samen met alle gerelateerde resources binnen
+        deze API.
+      parameters:
+      - name: Accept-Crs
```

Comment:
We'll need to look into this - these headers should not be required for DELETE requests

Reply:
New issue?

Reply:
yeah, new issue, I created https://github.com/VNG-Realisatie/gemma-zaken-common/issues/5